client/app/index.js

```diff
@@ -26,6 +34,8 @@ function loadCurrentModule() {
 }
 
 function loadModules() {
+  loadAceEditorDependencies();
```

Comment:
It could be placed at the top of `layout_ace_editors.js` too. Just wanted to put it where all the loading is done.